op-node/rollup/derive/span_batch_tx.go

```diff
@@ -69,7 +69,7 @@ func (tx *spanBatchTx) MarshalBinary() ([]byte, error) {
 }
 
 // setDecoded sets the inner transaction after decoding.
-func (tx *spanBatchTx) setDecoded(inner spanBatchTxData, size uint64) {
+func (tx *spanBatchTx) setDecoded(inner spanBatchTxData, _ /*size*/ uint64) {
```

Comment:
suggest just removing `/*size*/`, and perhaps making it a comment on the function that size is required for the interface, but unused (actually, *is* it required for the interface?)

Reply:
It seem unnecessary, I removed it .